tsfile/src/main/java/org/apache/iotdb/tsfile/read/TsFileSequenceReader.java

```diff
@@ -246,21 +246,21 @@ public Chunk readMemChunk(ChunkMetaData metaData) throws IOException {
   /**
    * not thread safe.
    *
-   * @param type -given tsfile data type
+   * @param type given tsfile data type
    */
   public PageHeader readPageHeader(TSDataType type) throws IOException {
-    return readPageHeader(type, -1);
+    return PageHeader.deserializeFrom(tsFileInput.wrapAsInputStream(), type);
   }
 
   /**
-   * notice, this function will modify channel's position.
+   * read the page's header.
    *
-   * @param position the file offset of this page header's header
+   * @param position the file offset of this chunk's header
+   * @param markerRead true if the offset does not contains the marker , otherwise false
+   * @param dataType given tsfile data type
    */
-  public PageHeader readPageHeader(TSDataType type, long position) throws IOException {
-    int size = PageHeader.calculatePageHeaderSize(type);
-    ByteBuffer buffer = readData(position, size);
-    return PageHeader.deserializeFrom(buffer, type);
+  private PageHeader readPageHeader(long position, boolean markerRead, TSDataType dataType) throws IOException {
```

Comment:
You'd better put dataType as the first parameter.

Reply:
By the way, who uses this method?

Reply:
fixed, this method is created in case someone intend to read a page header at specific position.